bedrock/firefox/templates/firefox/features/adblocker.html

```diff
@@ -24,133 +24,153 @@
+      {% trans url=url('firefox.features.private-browsing') %}
+      If ads don’t bother you and you don’t mind being followed by trackers and third-party cookies, then the Standard setting should work for you. To get trackers off your tail in Standard mode, use a <a href="{{ url }}">Private Browsing</a> window.
       {% endtrans %}
     </p>
   </section>
 
   <section class="mzp-l-content l-content-image">
-    {{ high_res_img('firefox/features/privacy-illustration2.png', {'alt': '', 'class': 'mzp-c-billboard-image'}) }}
+    {{ high_res_img('firefox/features/adblocker/content-blocking.png', {'alt': '', 'class': 'mzp-c-billboard-image'}) }}
+  </section>
+
+  <section class=" mzp-l-content l-content-narrow">
+    <h2 class="section-title">{{_('Get tough with Strict mode') }}</h2>
+    <p>
+      {% trans %}
+      If seeing too many ads ruins your day, then the Strict mode is a better fit. This mode will block known third-party trackers and cookies in all Firefox windows.
+      {% endtrans %}
+    </p>
   </section>
 
   <section class="mzp-l-content l-content-narrow">
-    <h2 class="section-title">{{ _('How to set Tracking Protection on') }}</h2>
-    <p>{{ _('With Tracking Protection and an ad blocker installed, you’re getting a safe and speedy web experience. Here’s how to get started:') }}</p>
-    <ol class="mzp-u-list-styled">
-      <li>{{ _('Click the menu button and choose Options (or Preferences on a Mac.)') }}</li>
-      <li>{{ _('Click Privacy & Security on the left and go to the Tracking Protection section.') }}</li>
-      <li>{{_('Choose between the following options:') }}</li>
-      <ul class="mzp-u-list-styled">
-        <li>
-          <span>{{_('Always:') }} </span>
-          {{_('Keep Tracking Protection on as you browse.') }}
-        </li>
-        <li>
-          <span>{{_('Only in private windows:') }} </span>
-          {{_('Turns Tracking Protection on only if you’re using Private Browsing.') }}
-        </li>
-        <li>
-          <span>{{_('Never:') }} </span>
-          {{_('Turns off Tracking Protection for all sites.') }}
-        </li>
-      </ul>
-    </ol>
+    <h2 class="section-title">{{_('Do-it-yourself Custom mode') }}</h2>
+    <p>
+      {% trans %}
+      The custom setting give you the ultimate choice. You to decide what you’re blocking and if you’re blocking trackers, cookies or both. If you allow cookies from a website, you’ll automatically be in Custom mode.
+      {% endtrans %}
+    </p>
   </section>
 
   <section class="mzp-l-content l-content-image">
-    {{ high_res_img('firefox/features/laptop.png', {'alt': '', 'class': 'mzp-c-billboard-image'}) }}
+    {{ high_res_img('firefox/features/adblocker/content-blocking-custom.png', {'alt': '', 'class': 'mzp-c-billboard-image'}) }}
```

Comment:
This screenshot is from Dev Edition and shows the Cryptominer and Fingerprinter blocking features that haven't landed in the release channel yet. The copy doesn't reference those new features and just says "blocking trackers, cookies or both" indicating only two options. 

We could just update the screenshot with one from Fx 66, but I'd also suggest tweaking the copy to be more future-proof since I think these new blocking features are scheduled for Fx 67.

Reply:
I took these screenshots in Fx 67, so I think we should keep it and maybe update the copy.
I will ping in the copy doc to include Cryptominer and Fingerprinter.

Reply:
@craigcook 
The copywriter, Aleks, brought up an excellent point:

>  I'm wondering if it would create confusion to bring cryptomining into the conversation? My instinct says people who are searching to find out what an ad blocker is won't know about crypto.

I am inclined to agree that maybe it would create confusion. Do you think this is a blocker?

Reply:
Well the real problem is that the copy and the screenshot don't match. The copy only covers "trackers, cookies or both" which implies two options when there are four. And I do think trying to call out cryptomining or fingerprinting will further complicate it, so maybe we don't need to mention them but could at least rephrase it to be more open-ended and not imply only two things can be blocked.

Maybe something like "You can decide what you’re blocking, including trackers, cookies and more." ?

Reply:
Aleks agreed with your suggested-- updated to "You can decide what you’re blocking, including trackers, cookies and more."